app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -217,59 +242,32 @@ span.preview_text_element_name {
   }
 }
 
-.element_foot {
-  padding: 2*$default-padding;
-  width: 100%;
-  height: 29px;
-  margin-left: -8px;
-  margin-right: -8px;
-  background-color: $medium-gray;
+.element-toolbar {
+  padding: 6px 0;
+  height: 6*$default-padding;
+  margin-bottom: 2*$default-margin;
+  border-bottom: 1px solid $medium-gray;
+  background-color: #f0f0f0;
 
   .element_tools {
     float: left;
-    margin-right: 8px;
-    margin-top: 6px;
+    margin-top: -3px;
+    margin-left: -4px;
   }
+}
 
-  .element_public {
-    float: left;
-    margin-top: 6px;
-  }
+.element-footer {
+  border-top: 1px solid $medium-gray;
+  margin: 8px 0 0 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `8px 0 0` instead of `8px 0 0 0`